cli/launchpad/draw.go

```diff
@@ -0,0 +1,141 @@
+package launchpad
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"os"
+)
+
+// diagram represents a GCP draw diagram
+type diagram struct {
+	paths []*diagramPath
+	group *diagramGroup
+}
+
+// diagramGroup represents a group of cards (including a top-level diagram)
+type diagramGroup struct {
+	d      *diagram
```

Comment:
I'm wondering if this is needed, when does a group need access its diagram? I see only on creating another group under the same diagram. Is this to pass some metadata?